pallets/delegation/src/lib.rs

```diff
@@ -273,6 +275,8 @@ pub mod pallet {
 		UnauthorizedRemoval,
 		/// The delegation creator is not allowed to create the delegation.
 		UnauthorizedDelegation,
+		/// The operation wasn't allowed because of insufficient rights.
+		AccessDenied,
```

Comment:
Should maybe this be part of the attestation crate? It is only used in the the access control, which requires the implementer to return a generic `DispatchError`, so maybe it would make sense to either declare it in the attestation pallet, or as a separate error?

Reply:
The point of defining it here is that the Delegation pallet could send more detailed errors. I Would actually prefer if we could give a reason for not allowing access.
If I understood separate error correctly, this would be something that cannot be shown to the user?

Reply:
I mean the dispatch error would still be bubbled up to the user, I guess. This is not used anywhere inside the pallet, and furthermore it is an attestation error, really, and not a delegation one.

Reply:
It is used in the delegation pallet. The implementation of `AttestationAccessControl` belongs to the delegation pallet

Reply:
Makes sense.